jvm/src/main/scala/doodle/jvm/DoodlePanel.scala

```diff
@@ -102,14 +100,26 @@ class DoodlePanel private(private var _image: Image) extends JPanel {
 
         paintImage(t, Point(origin.x, tOriginY), context)
         paintImage(b, Point(origin.x, bOriginY), context)
-      case At(vec, i) =>
+      case At(vec, i) ⇒
         paintImage(i, origin + vec, context)
 
-      case ContextTransform(f, i) =>
+      case ContextTransform(f, i) ⇒
         paintImage(i, origin, f(context))
 
-      case d: Drawable =>
+      case d: Drawable ⇒
         paintImage(d.draw, origin, context)
+      case _path @ Path(head, tail) ⇒
+        val box = BoundingBox(_path)
+        val path = new Path2D.Double()
+        path.moveTo(head.x, head.y)
+        tail.foreach {
+          case MoveTo(x, y)                        ⇒ path.moveTo(x, y)
+          case LineTo(x, y)                        ⇒ path.lineTo(x, y)
+          case ArcTo(x, y, x2, y2, r)              ⇒ path.curveTo(x, y, x2, y2, r, 0) //Wrong
```

Comment:
Wait to see if there is any interest in the pull request and I'll invest time in fixing.